openmdao/docs/usr-guide/tutorials/recording.rst

```diff
@@ -85,7 +85,7 @@ optimizer progresses.
 ::
 
     recorder = SqliteRecorder('paraboloid')
-    driver.add_recorder(recorder)
+    top.driver.add_recorder(recorder)
```

Comment:
Might want to put a doctest directive on these sections so that they run. that would prevent syntax errors. I'd recommend just doing it on the instantiation lines, but don't make the model run inside a doctest, since I don't know where the sql file will go or how to clean it up.